MCServer/Plugins/APIDump/APIDesc.lua

```diff
@@ -197,6 +197,7 @@ g_APIDesc =
 				baMetas = { Notes = "Operations should work on block metas" },
 				baLight = { Notes = "Operations should work on block (emissive) light" },
 				baSkyLight = { Notes = "Operations should work on skylight" },
+				msDifference = { Notes = "Block becomes air if Src and Dst are the same." },
```

Comment:
What happens to all the other blocks, are they kept or set to a different value or ...?